django/applications/catmaid/control/sampler.py

```diff
@@ -533,7 +533,7 @@ def delete_sampler(request, project_id, sampler_id):
 
             if nodes_to_remove:
                 query_parts = []
-                params = []
+                remove_params = []
```

Comment:
there was a different params variable higher up in this function

Reply:
:+1: